proposals/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "webb-proposals"
-version = "0.3.7"
+version = "0.3.8"
```

```suggestion
version = "0.4.0"
```
Since you changed the name of the `Proposal` to `ProposalTrait`, thus a breaking change.